ethers-contract/src/lib.rs

```diff
@@ -3,24 +3,25 @@
 #![warn(missing_docs)]
 #![cfg_attr(docsrs, feature(doc_cfg))]
 
-#[path = "contract.rs"]
-mod _contract;
-pub use _contract::{Contract, ContractInstance};
+macro_rules! if_providers {
```

Comment:
I'd prefer having `pub(crate) use` for this macro and importing/calling by full path on call site

Reply:
I want to avoid importing of this specific macro